### PR TITLE
Release 1.0

### PR DESCRIPTION
**Pull request by KGuillemot**
## Improvements/New features
 - File log forwarders : Add 'stock_as_raw', 'retention_time' and 'rotation_period' attributes
 - Logs rotation : Add LogRotate service configuration for log forwarders files rotation
 - Reputation : Merge reputation_contexts and Feed objects
 - Reputation : Refactor ReputationContexts downloads
 - Darwin : Update Darwin statuses according to project modifications
 - PacketFilter : Add possibility to disable max-rate and max-src by setting them to 0 in frontend
 - PacketFilter : Set PF limit state default to 500'000
 - PacketFilter : Exclude broadcast packets from PacketFilter logs if IPv4 configured
 - Frontend : Remove stock_logs_locally attribute
 - Frontend : Force timeout client & server to 0 for tcp log mode frontends
 - Frontend : Send Impcap logs to Redis via publishing for Darwin reconciliation
 - Rsyslog : Send impstats logs to Redis and to /var/log/pstats-sec each seconds, and to /var/log/pstats each minutes
 - Rsyslog : Remove reputation enrichment for PacketFilter logs, they are dropped not logged
 - Dashboard : Add internal Redis log forwarder for dashboard, and select-it by default in impcap frontends 
 - Network : Force gateway_enable and ipv6_gateway_enable to YES

## Fixes
 - Vultured : Fix stop routine to prevent long wait, and fix Darwin filters statuses according to Darwin manager
 - Frontend : Remove host and path acls in HAProxy conf in TCP mode
 - Rsyslog : Fix rsyslog templates and rulesets

### CODE REVIEW

File: vulture_os/services/templates/services/frontend_edit.html [hunk 1]
```diff
@@ -885,26 +886,29 @@ <h4 class="panel-title"><i class="icon fa fa-ban"></i> {% trans "Form errors" %}
...
-        $('.forwarder-tag').hide();
-      }
-      show_log_condition_failure();
-    });
-
-
     /* Show log_condition_failure depending on mode and ruleset */
     function show_log_condition_failure() {
       if( $('#id_mode').val() == "log" && ['raw_to_json', 'impcap'].indexOf($('#id_ruleset').val()) < 0 ) {
+        console.log("show")
```
Comment (frikilax) [inline vulture_os/services/templates/services/frontend_edit.html]: necessary ?

File: vulture_os/services/templates/services/frontend_edit.html [hunk 2]
```diff
@@ -885,26 +886,29 @@ <h4 class="panel-title"><i class="icon fa fa-ban"></i> {% trans "Form errors" %}
...
         $('.log-failure').show();
       } else {
         $('.log-failure').hide();
+        console.log("hide")
```
Comment (frikilax) [inline vulture_os/services/templates/services/frontend_edit.html]: necessary ?

File: vulture_os/services/templates/services/frontend_edit.html (hunk 1 above)
Comment (KGuillemot) [reply vulture_os/services/templates/services/frontend_edit.html]: Removed

File: vulture_os/services/templates/services/frontend_edit.html (hunk 2 above)
Comment (KGuillemot) [reply vulture_os/services/templates/services/frontend_edit.html]: Removed

**Pull request merged**